monero-outreach-docs/translations/es/archive/wallet_primer_es.md

```diff
@@ -0,0 +1,103 @@
+
+### _Base Técnica_
+
+Es importante tener un modelo mental de cómo funciona un monedero de papel para estar confiado y sentirse seguro al usarlo. La clave privada de gasto que mencionamos anteriormente, en su forma hexadecimal (usando del 0 al 9 y de la 'a' a la 'f'), se ve algo así:
+
+_Clave privada de gasto:_
+**09f8225e9f6dd95457a0bca31b66a599 6e6da8be8195025817c36bf490ae0903**
+
+Representa aproximadamente 256 bits de información aleatoria, contiene muchos caracteres y es tan complicado que nadie puede calcularlo. Puedes escribirla en algún lado y ocultarla, y entonces, serás dueño de los Monero que controla.
+
+Porque la clave privada de gasto sin procesar es difícil de manejar y propensa a errores, se han creado grupos de palabras con asignaciones regulares un poco más amigables llamadas frases mnemónicas. Las dos asignaciones más importantes para Monero son las siguientes:
+
+**Asignación 1 (25 palabras):** La asignación de palabras internas del monedero oficial de Monero usa 25 palabras seleccionadas de un conjunto de 1626 palabras. La asignación incluye restricciones para que la selección aleatoria de 25 palabras usualmente no sea válida. Esto le ayuda al software detectar errores. Puedes ver la lista de palabras en Inglés aquí: [github.com/monero-project/monero/blob/master/src/mnemonics/english.h](https://github.com/monero-project/monero/blob/master/src/mnemonics/english.h). La asignación de palabras para la dirección que mostramos anteriormente de la clave privada de gasto en frase mnemónica es: _{moon pram oust mime boil boat nutshell moisture ionic enjoy below ungainly loaded rage etched atom reduce almost glass vexed jaded coils pioneer pyramid nutshell}._
+
+**Asignación 2 (24 palabras):** La asignación estándar de la Propuesta 39 (BIP39) de la mejora de Bitcoin, es una asignación de propósito general usado por el monedero de hardware Ledger, por ejemplo. Está compuesto por 24 palabras ordenadas, seleccionadas de un conjunto de 2048. Tal como la Asignación 1, tiene restricciones internas para que la mayoría de combinaciones de palabras sean inválidas. La lista de palabras en Inglés pueden ser vistas aquí: [github.com/bitcoin/bips/blob/master/bip-0039/english.txt](https://github.com/bitcoin/bips/blob/master/bip-0039/english.txt). Un ejemplo de la frase mnemónica de este tipo es la siguiente: _{open birth pepper electric nuclear shine stable ritual napkin quarter aunt demand thing cable tuna lucky vehicle taste tourist grape buffalo fitness indoor draft}._
```

Comment:
**Sugerencia:** 
Parrafo completo:
"La propuesta de mejora estándar de Bitcoin 39 (BIP39). Este tipo común de conversión lo utiliza por ejemplo la billetera de hardware ledger. La conversión utiliza 24 palabras ordenadas, seleccionadas de un conjunto de 2048. Como en el caso de conversión 1, esta opción también tiene sus propias limitaciones internas, por lo que la mayoría de combinaciones de palabras no son válidas.

Reply:
I got your suggestion and got to this suggestion:

"La propuesta de mejora estándar de Bitcoin 39 (BIP39), es un tipo común de conversión que utiliza, por ejemplo, el monedero de hardware ledger. La conversión utiliza 24 palabras ordenadas, seleccionadas de un conjunto de 2048. Como en el caso de la conversión 1, esta opción también tiene sus propias restricciones internas, para que la mayoría de combinaciones de palabras no son válidas."

Reply:
Checked, fixed, leaving comment open.